### PR TITLE
Feat : Home page created in react

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
Home page created

### CODE REVIEW

File: src/components/home/Home.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                    <Card
+                        img={"./images/homepage/webdev.png"}
+                        productName={"The complete 2022 Fullstack Web Developer Course"}
+                        productOwner={"Arjun"}
+                        noOfRatings={4, 352}
+                        price={199}
+                    />
+                    <Card
+                        img={"./images/homepage/webdev.png"}
+                        productName={"The complete 2022 Fullstack Web Developer Course"}
```
Comment (anik31) [inline src/components/home/Home.js:39]: Suggestion - you can make a file for product-data containing array of objects and then map over it. This way you don't have to write the card component everytime and pass data

File: src/components/home/Home.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            <section className="top-categories">
+                <h2>Top Categories</h2>
+                <div className="categories-container">
+                    {
+                        categories.map((item, i) => <CardCategory
+                            key={i}
+                            img={item.img}
+                            category={item.category}
+                        />
+                        )
```
Comment (anik31) [inline src/components/home/Home.js:71]: Appreciation - this is the approach which should be followed for generating cards. You can do the same for the above product cards.

File: src/components/navbar/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+const Navbar = () => {
+  return (
+    <nav className="nav nav-boxshadow">
+        <a href="/" className="brand my-brand">
+            <div className="logo">
+                <img loading="lazy" src="./images/nav/nav-logo.png" alt="logo" />
+            </div>
```
Comment (anik31) [inline src/components/navbar/Navbar.js:9]: suggestion - don't use loading="lazy" in navbar as it is the first thing which will be displayed to the user. You can use it in places which the user will not see immediately.

File: src/components/template/Card.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+const Card = ({img,productName, productOwner, noOfRatings, price  }) => {
+    return (
```
Comment (anik31) [inline src/components/template/Card.js:4]: suggestion - you can also add an outOfStock prop which will store a boolean value & based on that you can change the css.

reference - https://neog-lms.vercel.app/docs/week-five/05-week-five-exercises#ex13-out-of-stock-grey

**Review by anik31 — COMMENTED** [review]
Nice work @arjun-computer-geek 💯 The code is very clean.

Suggestion - add some screenshots to the PR description as it will be easier for the reviewer to review the UI as well.

File: src/components/navbar/Navbar.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
+import React from 'react'
```
Comment (Developers-codz) [inline src/components/navbar/Navbar.js:1]: there is no need of importing react, you can remove that.

File: src/components/navbar/Navbar.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+            <a href="">
+                <i className="fa fa-shopping-cart"></i>
+                <span className="nav-icon-text">cart</span>
+            </a>
+        </div>
+    </nav>
+  )
+}
+
+export default Navbar
```
Comment (Developers-codz) [inline src/components/navbar/Navbar.js:39]: default export don't sound good, can make a common index.js file and importing from there will reduce no. of importing files line in app.js file

**Review by Developers-codz — APPROVED** [review]
Other than that everything looks good to me, Good work

File: src/components/navbar/Navbar.js (hunk 6 above)
Comment (arjun-computer-geek) [reply src/components/navbar/Navbar.js:39]: I have only one component that's why

**Pull request merged**